shlr/sdb/src/main.c

```diff
@@ -537,7 +537,6 @@ static int gen_gperf(const char *file, const char *name) {
 	char *out = malloc (strlen (file) + 32);
 	snprintf (out, strlen (file) + 32, "%s.gperf", name);
 	int wd = open (out, O_RDWR);
-	ftruncate (wd, 0);
```

Comment:
pls submit this fix into the sdb repo, not the copy in r2

Reply:
it's already fixed there